app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class FeedbacksController < ApplicationController
+  FEEDBACK_TYPES = %w[freetext satisfaction].freeze
+
+  def create
+    if params[:type].nil? || !FEEDBACK_TYPES.include?(params[:type])
+      raise "Feedback type needs to be specified"
```

Comment:
I'm not sure about this behaviour. If something tries to submit an invalid payload to this controller, I don't think we should return a 500 status code and trigger a sentry notification etc, unless we're sure this is a sign that our code is broken somewhere. Otherwise, I think returning a 422 or 400 feels more appropriate

Reply:
to chat about when pairing